### ISSUE TITLE
[Adaptive Cards iOS - FoodOrder]: 'How would you like your steak prepared?' option under Steak is not accessible through the swipe gesture. 

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6965]
https://microsoft.visualstudio.com/OS/_workitems/edit/33764423
